imports/ui/components/pwd-strength.js

```diff
@@ -1,42 +0,0 @@
-/* Password strength validator */
```

Comment:
Hmm right, well it is a solution for now, but I think we might separate it again in the future if we start adding to much logic in here